packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -336,15 +337,19 @@ const App = (props) => {
                                 <link
                                     rel="alternate"
                                     hrefLang={locale.id.toLowerCase()}
-                                    href={`${appOrigin}${buildUrl(location.pathname)}`}
+                                    href={`${appOrigin}${getPathWithLocale(locale.id, buildUrl, {
+                                        location
+                                    })}`}
```

Comment:
There's no need to import `getPathWithLocale`.

We can simply pass the parameters to the `buildUrl`:
```suggestion
                                    href={`${appOrigin}${buildUrl(location.pathname,  site.alias || site.id, locale?.alias || locale?.id)}`}
```